### DISCUSSION TITLE
Add sound pressure sensor device class

### DISCUSSION THREAD

**Discussion by epenet** [home-assistant/architecture#837] [Entity Models] [answered]
I propose that we add a `sound level` sensor device_class to home assistant core and frontend (+ documentation) with supported units:

- `dB`
- `dBa`

Looking at the use of the unit, these integrations would benefit: `airq`, `homekit_controller`, `isy994`, `mysensors`, `netatmo`, `point`

**Comment by epenet** [comment]
Implementation PR: https://github.com/home-assistant/core/pull/83453

**Comment by emontnemery** [comment]
I think it should be `dB(A)`, not `dBa`: https://en.wikipedia.org/wiki/A-weighting

**Reply by parautenbach** [reply]
dBA is also acceptable (also noted in the Wikipedia page). Common usage (from what I've seen) is dBA over dB(A). Definitely not dBa. 

For a really comprehensive list of dB variants see https://en.wikipedia.org/wiki/Decibel.

**Reply by parautenbach** [reply]
Can units have properties, e.g. so that the unit is dB, but there's an attribute/property to say it uses A-weighted values?

> dBA, dBB, and dBC
These symbols are often used to denote the use of different [weighting filters](https://en.wikipedia.org/wiki/Weighting_filter), used to approximate the human ear's [response](https://en.wikipedia.org/wiki/Stimulus_(psychology)) to sound, although the measurement is still in dB (SPL). These measurements usually refer to noise and its effects on humans and other animals, and they are widely used in industry while discussing noise control issues, regulations and environmental standards. Other variations that may be seen are dBA or [dB(A)](https://en.wikipedia.org/wiki/A-weighting). According to standards from the International Electro-technical Committee ([IEC 61672-2013](https://en.wikipedia.org/wiki/IEC_61672))[[52]](https://en.wikipedia.org/wiki/Decibel#cite_note-54) and the American National Standards Institute, [ANSI S1.4](https://en.wikipedia.org/w/index.php?title=ANSI_S1.4&action=edit&redlink=1),[[53]](https://en.wikipedia.org/wiki/Decibel#cite_note-55) the preferred usage is to write LA = x dB. Nevertheless, the units dBA and dB(A) are still commonly used as a shorthand for A‑weighted measurements. Compare [dBc](https://en.wikipedia.org/wiki/DBc), used in telecommunications.

**Reply by epenet** [reply]
The discussion here is more about the device class - less about the unit.
To discuss the unit, I think it would be better discussed on this PR: https://github.com/home-assistant/core/pull/83450

**Reply by parautenbach** [reply]
Of course, you're right. I'll post my comments there.

**Comment by emontnemery** [comment, marked as answer]
The proposal is accepted, the unit can be discussed in core PR https://github.com/home-assistant/core/pull/83450
